rewrite-maven/src/test/java/org/openrewrite/maven/ChangeDependencyGroupIdAndArtifactIdTest.java

```diff
@@ -1676,4 +1676,96 @@ void changeDependencyGroupIdAndArtifactIdWithVersionProperty() {
           )
         );
     }
+
+    @Test
+    void changeVersionPropertyInParentPom() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeDependencyGroupIdAndArtifactId(
+            "javax.activation",
+            "javax.activation-api",
+            "jakarta.activation",
+            "jakarta.activation-api",
+            "1.2.x",
+            null
+          )),
+          mavenProject("project",
+            //language=xml
+            pomXml(
+              """
+                <project>
+                  <groupId>com.mycompany.app</groupId>
+                  <artifactId>parent-project</artifactId>
+                  <version>1</version>
+                  <properties>
+                    <activation.api>1.2.0</activation.api>
+                  </properties>
+                  <modules>
+                    <module>sub-project</module>
+                  </modules>
+                </project>
+                """,
+              """
+                <project>
+                  <groupId>com.mycompany.app</groupId>
+                  <artifactId>parent-project</artifactId>
+                  <version>1</version>
+                  <properties>
+                    <activation.api>1.2.2</activation.api>
```

Comment:
```suggestion
                    <activation.api.groupid>javax.activation</activation.api.groupid>
                    <activation.api.artifact>javax.activation-api</activation.api.artifact>
                    <activation.api.version>1.2.0</activation.api.version>
```

should be able to work now as well! _But it doen't yet, so I'll work on some more..._